### PR TITLE
Edit User Guide: opencus, delcus

### PR DESCRIPTION
**Pull request by nvmdava**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -132,19 +146,20 @@ Examples:
...
 * Deletes the person at the specified `INDEX`.
 * The index refers to the index number shown in the displayed person list.
 * The index **must be a positive integer** 1, 2, 3, …​
 
 Examples:
-* `list` followed by `delete 2` deletes the 2nd person in the address book.
-* `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
+* `delcus 2`
+  Deletes the 2nd person in the ArtBuddy and all commissions made by the person.
+
```
Comment (seelengxd) [inline docs/UserGuide.md:162]:
```suggestion
```

Should this extra newline be removed?

**Review by seelengxd — CHANGES_REQUESTED** [review]
LGTM - minor nit to fix.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -91,6 +91,20 @@ Creates the customer entry for John Doe with his details including multiple tags
...
 Creates the customer entry for Betsy Crowe with her email and phone number.
 
+### Viewing a customer: `opencus` [coming soon]
+
+Opens a customer and shows customer details and switches tab to commissions to show commissions made by the customer.
+
+Format: `opencus INDEX`
+
+* Views the customer at `INDEX`.
+* The index refers to the index number shown in the displayed person list.
```
Comment (seelengxd) [inline docs/UserGuide.md:101]:
```suggestion
* The index refers to the index number shown in the displayed customer list.
```

File: docs/UserGuide.md [hunk 3]
```diff
@@ -132,19 +146,19 @@ Examples:
...
 
 * Deletes the person at the specified `INDEX`.
 * The index refers to the index number shown in the displayed person list.
 * The index **must be a positive integer** 1, 2, 3, …​
 
 Examples:
-* `list` followed by `delete 2` deletes the 2nd person in the address book.
-* `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
+* `delcus 2`
+  Deletes the 2nd person in the ArtBuddy and all commissions made by the person.
```
Comment (seelengxd) [inline docs/UserGuide.md:161]:
```suggestion
  Deletes the 2nd person in the ArtBuddy and all commissions made by the customer.
```

File: docs/UserGuide.md [hunk 4]
```diff
@@ -132,19 +146,19 @@ Examples:
...
+### Deleting a customer : `delcus` [coming soon]
 
-Deletes the specified person from the address book.
+Deletes the customer from the ArtBuddy.
 
-Format: `delete INDEX`
+Format: `delcus INDEX`
 
 * Deletes the person at the specified `INDEX`.
 * The index refers to the index number shown in the displayed person list.
```
Comment (seelengxd) [inline docs/UserGuide.md:156]:
```suggestion
* Deletes the customer at the specified `INDEX`.
* The index refers to the index number shown in the displayed customer list.
```

**Review by seelengxd — CHANGES_REQUESTED** [review]
LGTM (almost) - can we use `customer` for consistency? :P

**Comment by nvmdava** [comment]
True, thanks for noticing

**Review by seelengxd — APPROVED** [review]

**Pull request merged**